test/test_unit_user_logic.py

```diff
@@ -105,6 +121,7 @@ def test_rotate_personal_api_key_raises_exception_when_unauthorized(self):
     '''
     Raises exception when operation was not able to be completed successfully
     '''
+
```

Comment:
remove these lines between comments and test

Reply:
Removed